cypress/support/globals.js

```diff
@@ -39,6 +39,24 @@ export const CATEGORIES = {
   Stability: ['stability'],
 };
 
+export const pathwaysTableColumns = [
+  {
+    title: intl.formatMessage(messages.pathwaysName),
+  },
+  {
+    title: intl.formatMessage(messages.category),
+  },
+  {
+    title: intl.formatMessage(messages.systems),
+  },
+  {
+    title: intl.formatMessage(messages.reboot),
+  },
+  {
+    title: intl.formatMessage(messages.reclvl),
+  },
+];
+
```

Comment:
Why would you use `intl` in your test data? Why can't we use just an English string?

Reply:
Thanks for the suggestion. Included that